contributors/devel/development.md

```diff
@@ -247,6 +247,8 @@ _If you have upstream write access_, please refrain from using the GitHub UI for
 creating PRs, because GitHub will create the PR branch inside the main
 repository rather than inside your fork.
 
+When pushing commits to a pull request, don't force push. Force pushing can corrupt your pull request.
```

Comment:
Does your `push` the `first push` or `some update` for your PR?

For `first push`, I think that we cannot use `force push`.

But for `update push`, seems using `force push` works well, as I cannot `push` without `--force` when updating a PR.

How did you create your first push and the update push? Thanks.